src/vimeo/players/overlays/MobileVimeoOffScreenOverlay.module.css

```diff
@@ -0,0 +1,10 @@
+.offScreenOverlayWrapper {
+	--z2: token(z2);
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	pointer-events: none;
+	z-index: calc(var(--z2) + 1);
```

Comment:
In the styled-components version it was:
```ts
z-index: ${Number(WDSVariables.z7) + 2};
```
And I'd assume with the new approach it'd be:
```css
.offScreenOverlayWrapper {
	--z7: token(z7);
...
	z-index: calc(var(--z7) + 1);
}
```